todo_app/app.py

```diff
@@ -45,6 +49,7 @@ def _get_next_status(current_status: ToDoState) -> ToDoState:
         if current_status == ToDoState.DOING:
             return ToDoState.DONE
         
-        raise Exception("Item state does not have a successor")
+        logging.getLogger(LOGGLY_LOGGER).warning("Attempt made to update item that does not have a successive status.")

```

Comment:
This seems slightly more fiddly than it needs to be; could we define a `get_logger` function in our utility file that we import instead?
```python
def get_logger():
    return logging.getLogger(LOGGLY_LOGGER)
```

That way, we'd just need to call `get_logger` here, and we'd avoid the rest of our code needing to think about Loggly at all